modules/nw-sriov-create-object.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-create-object_{context}"]
+
+= Using the SR-IOV network operator to create the `sriovNetwork` object
+The following is an example definition of an `sriovNetwork` object. In this case Intel and Mellanox configurations are identical.
```

Comment:
You could end this sentence with a colon, because it is referencing the subsequent code block.

Reply:
Done.